examples/astro-stores-demo/src/components/store/ProductFilters.tsx

```diff
@@ -91,16 +91,16 @@ export const ProductFilters: React.FC<ProductFiltersProps> = ({
             availableMaxPrice,
             selectedMinPrice,
             selectedMaxPrice,
-            setSelectedMinPrice,
-            setSelectedMaxPrice,
+            setMinPrice,
+            setMaxPrice,
```

Comment:
Specifically here I think it's more clear to use the previous naming to not confuse with the availableMin/MaxPrice